From a49913317e51d607bf43249143eedffaaa64a5a8 Mon Sep 17 00:00:00 2001
From: Vasant Hegde <hegdevasant@linux.vnet.ibm.com>
Date: Mon, 28 Nov 2016 14:57:27 +0530
Subject: [PATCH 32/43] devtree: Use consistent output format

Use consistent output format (FRU# <value>).

This change is specific to Power Systems.

Signed-off-by: Vasant Hegde <hegdevasant@linux.vnet.ibm.com>
---
 src/core/device-tree.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/core/device-tree.cc b/src/core/device-tree.cc
index 1e49f3a..21bb6d9 100644
--- a/src/core/device-tree.cc
+++ b/src/core/device-tree.cc
@@ -428,7 +428,7 @@ static void add_chip_vpd(string path, string name,
         data->product = hw::strip(get_string("part-number"));
 
       if (exists("fru-number"))
-        data->product += " FRU #" + hw::strip(get_string("fru-number"));
+        data->product += " FRU# " + hw::strip(get_string("fru-number"));
 
       vpd.insert(std::pair<uint32_t, chip_vpd_data *>(chip_id, data));
     }
@@ -776,7 +776,7 @@ void add_memory_bank(string name, string path, hwNode & core)
     product = get_string("part-number");
     if(exists("fru-number"))
     {
-      product += " FRU#" + get_string("fru-number");
+      product += " FRU# " + get_string("fru-number");
     }
     if(product != "")
       bank.setProduct(hw::strip(product));
-- 
2.10.2

